### PR TITLE
Guard service

### PR DESCRIPTION
**Pull request by andrbo**

### CODE REVIEW

File: src/app/labels/labels.component.ts [hunk 1]
```diff
@@ -18,15 +18,17 @@ const COMMA = 188;
...
 
+  @Input() canEdit = true;
   @Input() disabled = false;
   @Input() type;
 
   public showAddLabelCard = false;
   public canUpdateLabel = false;
   private indexOfClickedLabel;
   private labels: Label[];
+  public canEditLabel: boolean;
```
Comment (maeb) [inline src/app/labels/labels.component.ts]: This is redundant

File: src/app/labels/labels.component.html [hunk 2]
```diff
@@ -8,7 +8,7 @@ <h1 *ngIf="type =='Selector'">
     </ng-container>
   </h1>
   <span class="fill-space"></span>
-  <button mat-mini-fab (click)="newLabelCard(true)">
+  <button mat-mini-fab (click)="newLabelCard(true)" *ngIf="!disabled">
```
Comment (maeb) [inline src/app/labels/labels.component.html]: canEdit

File: src/app/configurations/crawlhostgroupconfig/crawlhostgroupconfig-details/crawlhostgroupconfig-details.component.ts [hunk 3]
```diff
@@ -25,10 +26,22 @@ export class CrawlHostGroupConfigDetailsComponent implements OnChanges {
 
   form: FormGroup;
 
-  constructor(private fb: FormBuilder) {
+  constructor(private fb: FormBuilder,
+              private roleService: RoleService) {
     this.createForm();
   }
+  //
```
Comment (maeb) [inline src/app/configurations/crawlhostgroupconfig/crawlhostgroupconfig-details/crawlhostgroupconfig-details.component.ts]: Remove comments

File: src/app/labels/labels.component.ts (hunk 1 above)
Comment (andrbo) [reply src/app/labels/labels.component.ts]: removed

File: src/app/labels/labels.component.html (hunk 2 above)
Comment (andrbo) [reply src/app/labels/labels.component.html]: changed to canEdit

File: src/app/configurations/crawlhostgroupconfig/crawlhostgroupconfig-details/crawlhostgroupconfig-details.component.ts (hunk 3 above)
Comment (andrbo) [reply src/app/configurations/crawlhostgroupconfig/crawlhostgroupconfig-details/crawlhostgroupconfig-details.component.ts]: comments removed

**Pull request merged**